### PR TITLE
🚚 Remove package prefix and rename "core" to "common"

### PR DESCRIPTION
**Pull request by rhinodavid**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,14 +79,14 @@
...
+    "build:web": "yarn build:local-packages && yarn workspace @upswyng/web build",
     "clean": "node clean.js",
-    "heroku-postbuild": "yarn workspace @upswyng/upswyng-core build && yarn workspace @upswyng/upswyng-server build",
-    "lint": "tsc --noEmit && eslint \"packages/upswyng-*/src/**/*.{js,jsx,ts,tsx}\" --fix",
-    "lint:ci": "tsc --noEmit && eslint \"packages/upswyng-*/src/**/*.{js,jsx,ts,tsx}\" --quiet",
-    "test": "concurrently -n core,server,web -c yellow,blue,cyan \"yarn workspace @upswyng/upswyng-core test\" \"yarn workspace @upswyng/upswyng-server test\" \"yarn workspace @upswyng/upswyng-web test\"",
+    "heroku-postbuild": "yarn workspace @upswyng/common build && yarn workspace @upswyng/server build",
+    "lint": "tsc --noEmit && eslint \"packages/**/src/**/*.{js,jsx,ts,tsx}\" --fix",
+    "lint:ci": "tsc --noEmit && eslint \"packages/**/src/**/*.{js,jsx,ts,tsx}\" --quiet",
+    "test": "concurrently -n common,server,web -c yellow,blue,cyan \"yarn workspace @upswyng/common test\" \"yarn workspace @upswyng/server test\" \"yarn workspace @upswyng/web test\"",
```
Comment (jacobvenable) [inline package.json:89]: The `@upswyng` prefix needs removed here as well ya?

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,12 +1,11 @@
 {
```
Comment (jacobvenable) [inline .vscode/settings.json:1]: I hadn't realized that we had committed the vscode settings. Maybe this should be ignored.

**Comment by jacobvenable** [comment]
vscode settings is an aside to this so I'll go ahead and merge so I can update my current PR

File: .vscode/settings.json (hunk 2 above)
Comment (rhinodavid) [reply .vscode/settings.json:1]: There's literally a force include line in the gitignore.

I thought about this a bit and I think we should def include the `.vscode` stuff -- I think we should recommend VSCode in our getting started. Hopefully new people can just clone the repo, add the recommended extensions and be ready to go. It's also helpful to make sure the plugin settings match the lint rule settings.

I also have started putting launch scripts in `launch.json`. It's super easy to get the dev server up in debug mode.

**Pull request merged**